funnel/__init__.py

```diff
@@ -30,6 +30,7 @@
 assets['schedules.css'][version] = 'css/schedules.css'
 assets['schedules.js'][version] = 'js/schedules.js'
 assets['screens.css'][version] = 'css/screens.css'
+assets['truncate.js'][version] = 'js/libs/jquery-succinct.js'
```

Comment:
The file should be named `truncate` as well.

Reply:
Or if the plugin is named `succint`, so should the asset.

Reply:
Changed it to `succinct`